### PR TITLE
Develop

### LINKED ISSUE #72: Add troubleshooting guide documentation
## Description

We need to add a documentation on troubleshooting to our portal to support the engineers and to have a place where we can point the community to

## TODO

- [ ] Failed deployments
- [ ] Using Docker
- [ ] Common CLI errors

### PR DESCRIPTION
**Pull request by drewstaylor**
This PR updates the docs site to version 1.0.1

- Fixes #72
- Delivers "Page 3" task from #71
- Homepage updated with links to newly added pages (Troubleshooting, Debugging)

New pages added:
- https://archway-docs--pr75-develop-csymf647.web.app/docs/create/advanced/debug/
- https://archway-docs--pr75-develop-csymf647.web.app/docs/create/advanced/troubleshooting

### CODE REVIEW

File: docs/create/advanced/cli.md.disabled [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+sidebar_position: 1
+title: CLI
+---
+
+# What is [Archway Cli](https://www.npmjs.com/package/@archwayhq/cli) really?
```
Comment (santanaluiz) [inline docs/create/advanced/cli.md.disabled:6]: should we point to GH instead? Also would be similar to `archwayd` then 🤔

File: docs/create/advanced/troubleshooting.mdx [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+```bash
+cargo install cargo-generate --features vendored-openssl
+cargo install cargo-run-script
+```
+
+## Deployment Failures
+
+### CosmWasm Rust optimizer fails
+
+If you're having trouble with the `cosmwasm/rust-optimizer`, you may need to update Cargo.toml at the root of your project with another version of the optimizer.
```
Comment (santanaluiz) [inline docs/create/advanced/troubleshooting.mdx:59]: should we add the link to `cosmwasm/rust-optimizer` repo here?

**Review by santanaluiz — DISMISSED** [review]
Nice points you put together 💪

File: docs/create/advanced/troubleshooting.mdx (hunk 2 above)
Comment (drewstaylor) [reply docs/create/advanced/troubleshooting.mdx:59]: Noted 👍

**Pull request merged**